### PR TITLE
[CN-1326]: add supported platform versions page

### PR DESCRIPTION
**Pull request by kutluhanmetin**

### CODE REVIEW

File: docs/modules/ROOT/pages/kubernetes-platform-support.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- 1.26
+- 1.27
+- 1.28
+- 1.29
+- 1.30
+
+Every Hazelcast Platform Operator version is tested against these Kubernetes versions.
+
+== Supported OCP Versions
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/kubernetes-platform-support.adoc:15]:
```suggestion
== Supported OpenShift Versions
```

File: docs/modules/ROOT/pages/kubernetes-platform-support.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+= Supported Platform Versions
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/kubernetes-platform-support.adoc:1]: I think we can move this one to the Overview page:
https://docs.hazelcast.com/operator/latest/
before `Community support` section:
https://docs.hazelcast.com/operator/latest/#community-support

**Review by hasancelik — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -13,6 +13,29 @@ Hazelcast Platform Operator allows your development and DevOps teams to automate
...
+
+- 1.26
+- 1.27
+- 1.28
+- 1.29
+- 1.30
+
+Every Hazelcast Platform Operator version is tested against these Kubernetes versions.
+
+=== Supported OCP Versions
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/index.adoc:30]:
```suggestion
=== OpenShift Versions
```

File: docs/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -13,6 +13,29 @@ Hazelcast Platform Operator allows your development and DevOps teams to automate
...
+
+Hazelcast Platform Operator supports following OCP versions:
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/index.adoc:32]:
```suggestion
Hazelcast Platform Operator supports following OpenShift versions:
```

File: docs/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -13,6 +13,29 @@ Hazelcast Platform Operator allows your development and DevOps teams to automate
 
 This documentation assumes an intermediate knowledge of Kubernetes.
 
+== Supported Platform Versions
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/index.adoc:16]:
```suggestion
== Supported Versions
```

File: docs/modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -13,6 +13,29 @@ Hazelcast Platform Operator allows your development and DevOps teams to automate
...
+
+=== Supported Kubernetes Versions
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/index.adoc:18]:
```suggestion
=== Kubernetes Versions
```

**Review by hasancelik — CHANGES_REQUESTED** [review]

**Review by hasancelik — APPROVED** [review]

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

**Pull request merged**